test/e2e/emissary_test.go

```diff
@@ -0,0 +1,254 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/aws/eks-anywhere/internal/pkg/api"
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+const (
+	emissaryPackageName   = "emissary"
+	emissaryPackagePrefix = "test"
+)
+
+func runCuratedPackageEmissaryInstall(test *framework.ClusterE2ETest) {
+	test.SetPackageBundleActive()
+	packageFile := test.BuildPackageConfigFile(emissaryPackageName, emissaryPackagePrefix, EksaPackagesNamespace)
+	test.KubectlClient.WaitForJobCompleted(context.TODO(), kubeconfig.FromClusterName(test.ClusterName), "1m", "complete", "eksa-auth-refresher", EksaPackagesNamespace)
```

Comment:
Remove this as latest CLI build no longer run a one time job.

Reply:
Fixed